browser/permissions/sources.gni

```diff
@@ -0,0 +1,6 @@
+brave_browser_permissions_sources = [
+  "//brave/browser/permissions/permission_lifetime_manager_factory.cc",
+  "//brave/browser/permissions/permission_lifetime_manager_factory.h",
+]
+
+brave_browser_permissions_deps = [ "//base" ]
```

Comment:
you also need deps here for `//brave/components/permissions`, `components/permissions` and `components/keyed_service/content`